gst/nnstreamer/nnstreamer_plugin_api_filter.h

```diff
@@ -40,6 +40,15 @@
 #define ACCL_NPU_SRCN_STR  "npu.srcn" /** srcn hardware supported by nnfw */
 #define ACCL_NPU_SR_STR  "npu.sr"
 
+#define GST_TENSOR_FILTER_FRAMEWORK_V0 (0xDEAFDEAD00000000ULL)
+#define GST_TENSOR_FILTER_FRAMEWORK_V1 (0xDEAFDEAD00010000ULL)
+
+/**
+ * @brief Check the value of the version field of GstTensorFilterFramework
+ */
+#define checkGstTensorFitlerFrameworkVersion(value, version) \
+  ((0xDEAFDEAD00000000ULL | ((version) << 16)) == (value & 0xFFFFFFFFFFFF0000ULL))
+
```

Comment:
Let's add
```
#define GST_TENSOR_FILTER_API_VERSION_DEFINED (1)
#define GST_TENSOR_FILTER_API_VERSION_MIN (0)
#define GST_TENSOR_FILTER_API_VERSION_MAX (1)
```

So that obsolete external (subplugins implemented in other git repos) code may detect it at build time.

Reply:
Updated. However, currently set `GST_TENSOR_FILTER_API_VERSION_DEFINED` to `0`, and `GST_TENSOR_FILTER_API_VERSION_MAX` to `0`.
Will upgrade these  to 1 once tensor_filter.c supports the new subplugin API.